src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -320,6 +321,15 @@ public final class Guardrails implements GuardrailsMBean
         replicaDiskUsage.minNotifyIntervalInMs(minNotifyInterval);
     }
 
+    /**
+     * Guardrail disabling repairs when there are mixed versions
+     */
+    public static final DisableFlag mixedRepairsEnabled =
```

Comment:
Curious why using `DisableFlag` in 4.1 and `EnableFlag` on trunk?

Reply:
thats a change in guardrails, in 4.1 it was disableflag, in trunk they changed it to enableflag and disableflag doesnt exist anymore